packages/shared/ReactErrorUtils.js

```diff
@@ -58,6 +59,13 @@ const ReactErrorUtils = {
   },
 
   /**
+   * Wrap a callback in whatever logic it needs. Used in FB to polyfill Promises.
+   */
+  wrapEventListener: function<T>(name: string, callback: T): T {
+    return wrapEventListener(name, callback);
+  },
```

Comment:
@gaearon I built this the same way as the other injection since they're related. Feel free to split them out when we move to static injection. At the very least it seems like we could move to something like what `fbjs/lib/EventListener` used to be.